components/calendar/src/gregorian.rs

```diff
@@ -228,8 +228,26 @@ pub(crate) fn year_as_gregorian(year: i32) -> types::FormattableYear {
     } else {
         types::FormattableYear {
             era: types::Era(tinystr!(16, "bce")),
-            number: 1 - year,
+            number: 1_i32.saturating_sub(year),
             related_iso: None,
         }
     }
 }
+
+
+#[cfg(test)]
+
```

Comment:
Remove blank line between `#[...]` line and the following line that it applies to.